### ISSUE TITLE
Whenever I Click edit It clears out the values for that ID I want to edit and leaves it as NULL

### ISSUE THREAD

**Issue by mustafehassan00** [PrimeAcademy/biscayne-syllabus#14]
## Description

**Your Real Name**

Mustafe Hassan 

**What do you want to happen?**

when I click edit It keeps the data in the database 

**What is actually happening?**

Whenever I Click edit It clears out the values for that ID I want to edit and leaves it as NULL

## Screenshots / Code Snippets

<img width="721" alt="Screenshot 2024-05-29 at 3 14 17 PM" src="https://github.com/PrimeAcademy/biscayne-syllabus/assets/155122726/09053cf2-9d0b-402a-a5ea-43efeac436bc">
<img width="721" alt="Screenshot 2024-05-29 at 3 14 17 PM" src="https://github.com/PrimeAcademy/biscayne-syllabus/assets/155122726/09053cf2-9d0b-402a-a5ea-43efeac436bc">
<img width="721" alt="Screenshot 2024-05-29 at 3 14 17 PM" src="https://github.com/PrimeAcademy/biscayne-syllabus/assets/155122726/09053cf2-9d0b-402a-a5ea-43efeac436bc">

## What did you try? Who did you ask?

I first asked blackbox.ai to see why is it clearing out 

I then could not conclude the answer as I made sure to double check to see if the data is being obtained 

I first asked blackbox.ai to see why is it clearing out 

I then could not conclude the answer as I made sure to double check to see if the data is being obtained 

## What branch is your code on? Did you git push?

main

## When was the last time you took a break?

5 minutes ago

**Comment by matthew-black** [comment]
Heyo @mustafehassan00!

Take a quick break, then come back and ask some other human being for a hand! That's the process with these issues.

After that, if you're still stuck, you'll need to update this issue to communicate what is currently working. Example:
* I have confirmed that my component's `onClick` calls `someFunctionName`.
* I have confirmed that `someFunctionName` has access to `whateverIdValue`.
* I have confirmed that `whateverIdValue` is being correctly dispatched.
* I have confirmed that the `editSomeThing` Saga function is receiving the correct id value.
* ...
